demo/admin/src/common/MasterMenu.tsx

```diff
@@ -45,7 +46,13 @@ export const pageTreeDocumentTypes: Record<string, DocumentInterface<any, any>>
     Page,
     Link,
 };
-const RedirectsPage = createRedirectsPage({ customTargets: { news: NewsLinkBlock }, scopeParts: ["domain"] });
+
+const customTargets = {
+    news: NewsLinkBlock,
+};
+export const RedirectLinkBlock = createRedirectLinkBlock(customTargets);
+// TODO: instead of passing customTargets here, we should use the createRedirectLinkBlock function to create a custom link block for the redirects. Needs a rework in the library before this is possible.
+const RedirectsPage = createRedirectsPage({ customTargets, scopeParts: ["domain"] });
```

Comment:
Doing all that in MainMenu is odd, I would move this code into a new `src/redirects/RedirectsPage.tsx` file.

(before it was a one-liner, which was kind of Ok)

Reply:
👍 
changed in: 318b871de3979338c99df8122f480ce030705039